### PR TITLE
chore(website): add info for the general options

### PR DESCRIPTION
**Pull request by SumantxD**
this PR tries to fix task 1 of the issue #1392

### CODE REVIEW

File: modelina-website/src/components/playground/options/GeneralOptions.tsx [hunk 1]
```diff
@@ -97,8 +97,15 @@ class GeneralOptions extends React.Component<
...
         </li>
 
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Include change type mapping example :">
+            <p>
+              In code generation, a common task is to map the data types from the input model to the output code. One approach to accomplish this is by using type mapping.
+              <br/><br/>
+              'Change type mapping' refers to the process of transforming one data type to another data type. This is a common technique in programming when data types need to be converted to make them compatible with another system or process
```
Comment (jonaslagoni) [inline modelina-website/src/components/playground/options/GeneralOptions.tsx:105]: I think we are saying the same thing twice, maybe this makes more sense?
```suggestion
              This option includes a simple example type mapping, that maps integers to a custom type.
```

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
Got a few suggestions to improve the wording 🙂 

Other then that it looks good 👌

File: modelina-website/src/components/playground/options/GeneralOptions.tsx [hunk 2]
```diff
@@ -97,8 +97,15 @@ class GeneralOptions extends React.Component<
           </label>
         </li>
 
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Include change type mapping example :">
+            <p>
+              In code generation, a common task is to map the data types from the input model to the output code. One approach to accomplish this is by using type mapping.
```
[the remark this answers is not in the document]
Comment (SumantxD) [reply modelina-website/src/components/playground/options/GeneralOptions.tsx]: done!! :tada:

File: modelina-website/src/components/playground/options/GeneralOptions.tsx (hunk 1 above)
Comment (SumantxD) [reply modelina-website/src/components/playground/options/GeneralOptions.tsx]: done!! :tada:

File: modelina-website/src/components/playground/options/GeneralOptions.tsx [hunk 3]
```diff
@@ -128,8 +140,25 @@ class GeneralOptions extends React.Component<
             />
           </label>
         </li>
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Change property naming format :">
+            <p>
+              This option allows you to customize the naming style for properties in your code. It provides several options to choose from:
```
[the remark this answers is not in the document]
Comment (SumantxD) [reply modelina-website/src/components/playground/options/GeneralOptions.tsx]: done!! :tada:

File: modelina-website/src/components/playground/options/GeneralOptions.tsx [hunk 4]
```diff
@@ -149,8 +178,25 @@ class GeneralOptions extends React.Component<
             />
           </label>
         </li>
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Change model naming format :">
+            <p>
+              This option allows you to customize the naming style for properties in your code. There are no limitations to how you can format it, but for this simple example it provides the following options:
```
[the remark this answers is not in the document]
Comment (SumantxD) [reply modelina-website/src/components/playground/options/GeneralOptions.tsx]: fixed !! :tada:

File: modelina-website/src/components/playground/options/GeneralOptions.tsx [hunk 5]
```diff
@@ -170,8 +216,25 @@ class GeneralOptions extends React.Component<
             />
           </label>
         </li>
-        <li>
-          <label className="flex items-center py-2 justify-between cursor-pointer">
+        <li className=' flex items-center'>
+          <InfoModal text="Change enum key naming format :">
+            <p>
+              This option allows you to customize the naming style for properties in your code. There are no limitations to how you can format it, but for this simple example it provides the following options:
```
[the remark this answers is not in the document]
Comment (SumantxD) [reply modelina-website/src/components/playground/options/GeneralOptions.tsx]: fixed !! :tada:

**Review by jonaslagoni — APPROVED** [review]
Nice 💪

**Pull request merged**